notice/store.go

```diff
@@ -78,3 +83,58 @@ func (s *Store) FindAllPolicyNotices(ctx context.Context, policyID string) ([]No
 
 	return notices, nil
 }
+
+// FindAllServiceNotices returns any relevant notices for the given service. Currently returns
+// notices pertaining to the system limit for unacknowledged alerts.
+func (s *Store) FindAllServiceNotices(ctx context.Context, serviceID string) ([]Notice, error) {
+	err := permission.LimitCheckAny(ctx, permission.User)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validate.UUID("serviceID", serviceID)
+	if err != nil {
+		return nil, err
+	}
+
+	count, err := gadb.New(s.db).CountUnackedAlertsByService(ctx, uuid.MustParse(serviceID))
+	if err != nil {
+		return nil, err
+	}
+
+	ls, err := limit.NewStore(ctx, s.db)
```

Comment:
Don't create a new store -- this will be inefficient and prepare all statements for every invocation. Pass the existing store, or add a query to `queries.sql`.

Maybe update the query for the use case:
```sql
-- name: NoticeUnackedAlertsByService :one
SELECT
    count(*),
    (
        SELECT
            max
        FROM
            config_limits
        WHERE
            id = 'unacked_alerts_per_service')
FROM
    alerts
WHERE
    service_id = $1::uuid
    AND status = 'triggered';

```

Reply:
^ that way we only make a single call to the DB and it would simplify the code in the notice package.